scene/gui/color_picker.cpp

```diff
@@ -857,10 +976,17 @@ void ColorPicker::add_preset(const Color &p_color) {
 		_add_preset_button(_get_preset_size(), p_color);
 	}
 
+	if (palette_name->get_text().size()) {
+		palette_name->set_text(vformat("%s*", palette_name->get_text().replace("*", "")));
```

Comment:
What is the `replace()` for? If you want to remove ending `*`, use `trim_suffix("*")`.

Reply:
I'd also suggest `(*)` to match the scene tabs in the editor as well as the script editor

Reply:
I don't think it is related to the editor or script editor, and it might unnecessarily confuse users who keep pressing Ctrl+S to save changes, but the color picker doesn't respond and keeps showing the (*) on these editors.

Reply:
I don't see how that'd be any more or less confusing because of that